vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -80,23 +85,28 @@ def remove_continuous_delivery(self):
         # TODO: this would be called by appservice web source-control delete
         return
 
-    def setup_continuous_delivery(self, azure_deployment_slot, app_type_details, vsts_account_name, create_account,
-                                  vsts_app_auth_token):
+    def setup_continuous_delivery(self, azure_deployment_slot, app_type_details, cd_project_url, create_account,
```

Comment:
Can you rename azure_deployment_slot to swap_with_slot or something to indicate that this is not the slot on which the cd is configured